packages/@types/function-tree/Path.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Payload } from "./primitives";
+
+// Is 'path' correct ?
+declare interface IPath {
```

Comment:
By the 'correct' I mean if the type of the prop is correct. In this case 'is string the correct type for path ?'. I will update questions to be more precise.